src/ExportPro.Common/ExportPro.Common.Shared/Library/BadRequestApiServiceResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net;
+
+
+namespace ExportPro.Common.Shared.Library
+{
+    public class BadRequestApiServiceResponse : ApiServiceResponse
+    {
+        public BadRequestApiServiceResponse(string message = null, string errorCode = null, List<string> validationErrors = null)
+        {
+            ApiState = HttpStatusCode.BadRequest;
+            Message = message;
+            ValidationErrors = validationErrors;
+            ErrorCode = errorCode ?? HttpStatusCode.BadRequest.ToString();
```

Comment:
Why do we need this errorCode in params ? we will have all time BadRequest for Error Code